composer.json

```diff
@@ -28,5 +28,17 @@
         "psr-4": {
             "Facebook\\InstantArticles\\": "tests/Facebook/InstantArticles/"
         }
+    },
+    "scripts": {
+        "all": [
+            "composer cs",
+            "composer test"
+        ],
+        "cs": [
+            "vendor/bin/phpcbf --standard=phpcs.xml -p || vendor/bin/phpcs --standard=phpcs.xml -p"
```

Comment:
We can take advantage of the PATH magic mentioned [here](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands) and simply use:

```
phpcbf --standard=phpcs.xml -p || phpcs --standard=phpcs.xml -p
```